blocks/gmo-campaign-details/gmo-campaign-details.js

```diff
@@ -189,11 +215,23 @@ export default async function decorate(block) {
         });
     });
     decorateIcons(block);
-    buildChannelScope(await deliverables, block);
-    buildDeliverablesTable(await deliverables, block);
+    buildChannelScope('deliverable-type',getDeliverableScopes(deliverables), block);
```

Comment:
as same function is being used for `deliverable-type` and `platforms`, we can rename function something like `buildUniqueScopes`